src/pages/iou/request/step/IOURequestStepDistance.js

```diff
@@ -94,6 +93,18 @@ function IOURequestStepDistance({
         scrollViewRef.current.scrollToEnd({animated: true});
     }, [numberOfPreviousWaypoints, numberOfWaypoints]);
 
+    const duplicateWaypointsError = useMemo(() => {
+        return _.keys(waypoints).length > 2 && _.size(validatedWaypoints) !== _.keys(waypoints).length;
```

Comment:
To avoid error while entering the first waypoint, we need to use nonEmptyWaypoints instead of waypoints
```
const nonEmptyWaypointsCount = _.filter(_.keys(waypoints), (key) => !_.isEmpty(waypoints[key])).length;
```

Reply:
To avoid error while entering the first waypoint, I inserted `isDisabled` prop by `nonEmptyWaypointsCount` in the "Next" button. So when editing the waypoints which mean the only one waypoint exists or empty, the button is disabled.